_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,130 @@
+    </ul>
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker run -d \
+              --env COCKROACH_DATABASE={DATABASE_NAME} \
+              --env COCKROACH_USER={USER_NAME} \
+              --env COCKROACH_PASSWORD={PASSWORD} \
+              --name=roach-single \
+              --hostname=roach-single \
+              -p 26257:26257 -p 8080:8080 \
+              -v "roach-single:/cockroach/cockroach-data" \
+              -v "~/init-scripts:/docker-entrypoint-initdb.d" \
+              cockroachdb/cockroach:latest start-single-node
+    ~~~
+
+    By default, a `certs` directory is created and CockroachDB starts in secure mode. To prevent this, add the `--insecure` flag after the `start-single-node` sub-command.
+
+    {{site.data.alerts.callout_info}}
+    The `COCKROACH_DATABASE`, `COCKROACH_USER`, and `COCKROACH_PASSWORD` environment variables and the contents of the `/docker-entrypoint-initdb.d` directory are ignored if you use `cockroach start` rather than `cockroach start-single-node`. They are also ignored if data exists in the `/cockroach/cockroach-data` directory within the container.
+    {{site.data.alerts.end}}
+
+    Docker adds a DNS entry that resolves the hostname `roach-single` to the container's IP address in Docker's default network. The following examples use this hostname.
+
+1. After the cluster is initialized, the cluster node prints helpful [startup details](cockroach-start.html#standard-output) to its log, including the DB Console URL and the SQL connection string. To retrieve `roach-single`'s startup details:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker exec -it roach-single grep 'node starting' cockroach-data/logs/cockroach.log -A 11
+    ~~~
+
+    ~~~ shell
+    CockroachDB node starting at 2022-09-28 14:35:27.495508396 +0000 UTC m=+1.370085757 (took 0.5s)
+    build:               CCL {{ page.page_version }} @ 2022/09/26 18:49:07 (go1.19.1)
+    webui:               https://172.18.0.3:8080
+    sql:                 postgresql://root@172.18.0.3:26257/defaultdb?sslcert=certs%2Fclient.root.crt&sslkey=certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=certs%2Fca.crt
+    sql (JDBC):          jdbc:postgresql://172.18.0.3:26257/defaultdb?sslcert=certs%2Fclient.root.crt&sslkey=certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=certs%2Fca.crt&user=root
+    RPC client flags:    /cockroach/cockroach <client cmd> --host=172.18.0.3:26257 --certs-dir=certs
+    logs:                /cockroach/cockroach-data/logs
+    temp dir:            /cockroach/cockroach-data/cockroach-temp2611102055
+    external I/O path:   /cockroach/cockroach-data/extern
+    store[0]:            path=/cockroach/cockroach-data
+    storage engine:      pebble
+    clusterID:           60f29a4e-1c87-4b0c-805d-eb73460766b1
+    status:              initialized new cluster
+    nodeID:              1
+    ~~~
+
+### Step 3. Connect to the cluster
+
```

Comment:
The formatting of some substeps is broken.  The code under substep 2 needs to be indented 1 more space (it's 3 spaces and all the other code snippets are indented 4 spaces) so that that the final step renders as 3.

<img width="1052" alt="Screen Shot 2022-09-30 at 7 05 41 AM" src="https://user-images.githubusercontent.com/20482960/193288092-8c738e11-65e8-4163-8dc3-bda0a3557d4d.png">